process/nodeStatusProcessor.go

```diff
@@ -475,8 +475,8 @@ func (nsp *NodeStatusProcessor) GetGasConfigs() (*data.GenericAPIResponse, error
 		return nil, err
 	}
 
+	var responseGenesisNodesConfig data.GenericAPIResponse
```

Comment:
rename to responseGasConfigs or something similar

Reply:
renamed